### PR TITLE
I got carried away with sorting things

### PR DESCRIPTION
**Pull request by bbannon**
Move doclist.php back to the front

### CODE REVIEW

File: events_ajax.php [hunk 1]
```diff
@@ -31,10 +31,10 @@
 require_once 'includes/validate.php';
 
 // Load Doc classes for attachments and comments
-require_once 'includes/classes/Doc.php';
 require_once 'includes/classes/DocList.php';
```
Comment (craigk5n) [inline events_ajax.php:34]: I believe we had a bug report about this order.  DocList requires Doc before it, right?

**Comment by bbannon** [comment]
Does it? I didn't see an 'extends Doc.php' anywhere. Maybe I missed it.

**Comment by craigk5n** [comment]
The problem in DocList.php is this line:
```
    $sql = Doc::getSQL ( $event_id, $type );
```
This requires Doc.php to be loaded first.

**Comment by bbannon** [comment]
Aha. I'm not really up on how classes work.

I think there are only a couple of places where doc.php should be relocated.

**Comment by bbannon** [comment]
Would it be be better, or work at all, if doclist.php required doc.php?

**Pull request closed without merging**